<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkUserInfoWidget</class>
 <widget class="QWidget" name="MarkUserInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MarkUserInfoWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget
{
	font-family:&quot;微软雅黑&quot;;
}


QLineEdit{
    background-color: #FFFFFF;
    border:none;
}
QLineEdit:focus{
    background-color: #FFFFFF;
    border-width: 1px;
    border-style: solid;
    border-color: #1BA9BA;
}

QLineEdit:disabled{
    background-color: #ECECEC;
    border:none;
}

QTextEdit{
    background-color: #FFFFFF;
    border:none;
}
QTextEdit:focus{
    background-color: #FFFFFF;
    border-width: 1px;
    border-style: solid;
    border-color: #1BA9BA;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="dialogborderback" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="TitlebarWidget" name="titlebar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>备注名称</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="lbMarkInfo">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>70</y>
           <width>201</width>
           <height>121</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>备注信息</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>10</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>用户ID</string>
         </property>
        </widget>
        <widget class="QLabel" name="lbConversationId">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>10</y>
           <width>201</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>备注信息</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lbMarkTitle">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>40</y>
           <width>201</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>备注名</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="btnOk">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>200</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>好</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitlebarWidget</class>
   <extends>QWidget</extends>
   <header>TitlebarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
